libs/autofile/group.go

```diff
@@ -122,11 +150,11 @@ func (g *Group) Close() {
 	g.mtx.Unlock()
 }
 
-// SetHeadSizeLimit allows you to overwrite default head size limit - 10MB.
-func (g *Group) SetHeadSizeLimit(limit int64) {
+// GroupCheckDuration returns the current groupCheckDuration.
+func (g *Group) GroupCheckDuration() time.Duration {
```

Comment:
unused?

Reply:
yes,  as HeadSizeLimit, expose only for check the current config.

Reply:
can you remove it then?

Reply:
done!